source/docs/romi-robot/programming-romi-in-labview.rst

```diff
@@ -0,0 +1,37 @@
+Programming the Romi
+====================
+
+Writing a LabVIEW program for the Romi is very similar to writing a program for a regular FRC robot. In fact, all the same tools (LabVIEW, Driver Station, SmartDashboard, etc) can be used with the Romi.
+
+Creating a Romi Program
```

Comment:
I will take a look at this, but the two words have different meanings. The LabVIEW project is the overarching structure that captures your code and related elements/settings/etc. The LabVIEW program is the code you have written to achieve the function. Many IDEs use this paradigm of the project/workspace/solution and I believe common usage is to still say you are "writing a program" not writing a "solution" or similar.

Reply:
I have used LabVIEW a lot and am very familiar.  I think there may be confusion about what I was highlighting here, I was referring to "Creating a Romi Program" which I would argue should be "Creating a Romi Project" the sentence above that using program is perfectly fine.  Most people I know however would say: "I do LabVIEW programming", "I write LabVIEW VIs" (rather than referring to the actual code as a program).

Reply:
Oops, I thought you were referring to the usage in the text above that. Agree that the heading could be tweaked.